app/experimenter/nimbus-ui/src/components/AppLayoutSidebarLaunched/index.stories.tsx

```diff
@@ -50,7 +52,8 @@ storiesOf("components/AppLayoutSidebarLaunched", module)
           overall: {},
           other_metrics: mockAnalysis().other_metrics,
         }}
-        {...{ experiment }}
+        experiment={experiment}
+        refetch={async () => {}}
```

Comment:
Since this is an empty function and it's repeated in a bunch of spots, we should either probably make it optional and just supply `async () => {}` as the prop default, or.. just make it optional and not call `refetch` if it's undefined?